tox.ini

```diff
@@ -32,22 +32,15 @@ commands =
 [testenv:fmt]
 description = Apply coding style standards to code
 deps =
-    isort
-    autopep8
+    -r{toxinidir}/requirements-dev.txt
```

Comment:
I can't comment on the correct line, but let's remove the `PYTHONBREAKPOINT=ipdb.set_trace` line above, now that ipdb is gone.